src/hotspot/cpu/x86/vm_version_x86.cpp

```diff
@@ -1031,7 +1038,7 @@ void VM_Version::get_processor_features() {
           FLAG_SET_DEFAULT(UseAESCTRIntrinsics, false);
         }
       } else {
-        if (supports_sse4_1()) {
+        if (UseSSE > 3) {
```

Comment:
We need  an other RFE to clean this up. We should disable CPU features according UseSSE value as we already do for UseAVX.
For these changes this is fine.

Reply:
It's already the case for `UseSSE` as well:
https://github.com/openjdk/jdk/blob/master/src/hotspot/cpu/x86/vm_version_x86.cpp#L858

Reply:
I find the original check clearer: it says that `UseAESCTRIntrinsics`-related code depends specifically on sse4.1. Please, leave it as is.

Reply:
Right, I first figured it was ignoring user settings, but as you point out the feature bits will be zeroed out anyhow.